src/modules/token/fungible/components/SendForm/SendForm.js

```diff
@@ -214,7 +222,7 @@ const SendForm = (props) => {
                   onChange={(value) => setRecipientChain(value)}
                   select={(selectedValue, option) => selectedValue?.chainID === option.chainID}
                 >
-                  {applications.map((application) => (
+                  {[mainChainApplication, ...applications].map((application) => (
```

Comment:
Are there some states where  mainChainApplication can be undefined? If line 79 fails to find an application?
In that case the first iteration will make `application` undefined and the app will crash. 
maybe 
`const allApplications = mainChainApplication ? [mainChainApplication, ...applications] : [...applications]`

Reply:
actually main chain can't be undefined because if its undefined it means application bootstrap would fail.